stable/datadog/templates/container-process-agent.yaml

```diff
@@ -37,8 +37,8 @@
       mountPath: /host/proc
       readOnly: true
     {{- if .Values.datadog.criSocketPath }}
-    - name: runtimesocket
-      mountPath: {{ .Values.datadog.criSocketPath }}
+    - name: runtimesocketpath
+      mountPath: /host{{ dir .Values.datadog.criSocketPath }}
```

Comment:
should we do  /host/{{ dir .Values.datadog.criSocketPath }} instead?

Reply:
You mean, with a `/` between `/host` and the `{{ dir … }}` ?
In that case, I would need add a call to `clean` to remove the double `/` when `criSocketPath` is an absolute path that already starts with a `/`.
Let me try that.

Reply:
Change done !